piperider_cli/compare_report.py

```diff
@@ -648,7 +651,7 @@ def _report_validater(answers, current) -> bool:
             else:
                 return len(current) == limit
 
-        profiler_outputs = self.list_existing_outputs()
+        profiler_outputs = self.profiler_outputs if self.profiler_outputs else self.list_existing_outputs()
```

Comment:
found a problem here when there are no cloud reports on the cloud, it tries to list existing local reports
1. it should not list local reports when for user when exec `piperider cloud compare-reports`
2. exception happened when `profiler_output_path` is not specified in listing existing outputs

Reply:
Fixed